docs/troubleshoot/security.md

```diff
@@ -2,6 +2,64 @@
 
 Refer to vulerability management and release documentation [here](https://github.com/ratify-project/ratify/blob/dev/SECURITY.md).
 
+## Signature Validation
+
+Ratify signs all dev images and dev helm OCI artifacts with Notary Project and Sigstore Cosign signatures.
+
+### Verifying Notary Project Signature
+
+The public certificate for verification can be found at `ratify.dev/.well-known/ratify-verification.crt`
+
+```shell
+curl ./ratify-verification.crt # TODO update this with link when PR is merged in ratify repo
```

Comment:
curious how long will the cert keep valid? do we need to rotate it after expiry?

Reply:
It's currently set to be valid for 1 year (expires 6/10/25) and it will auto renew after 9 months.